.github/workflows/build-test.yaml

```diff
@@ -128,6 +128,10 @@ jobs:
         - rust: 1.68
 
     steps:
+    - name: Remove unnecessary files
+      run: |
+        sudo rm -rf /usr/share/dotnet
+        sudo rm -rf "$AGENT_TOOLSDIRECTORY"    
```

Comment:
Wouldn't it better to just invalidate the cache?

That's probably the reason for these problems since both the old and new toolchains are now cached.

Reply:
I had the same issue in https://github.com/Concordium/concordium-node/pull/1118 which was fixed by clearing the cache of old junk.